crawl-ref/source/melee-attack.cc

```diff
@@ -1115,6 +1115,8 @@ class AuxBite: public AuxAttackType
 
         if (you.get_mutation_level(MUT_ACIDIC_BITE))
             return fang_damage + str_damage;
+        if (you.get_mutation_level(MUT_DRAIN_BITE))
+            return fang_damage + str_damage;
 
```

Comment:
Remove both of these `if`s. They do nothing, because `fang_damage + str_damage` is returned anyway.